src/style.css

```
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.big-search {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

move `font-family` to the `html` tag selector and make inputs inherit it 
```suggestion
  font-family: inherit;
```
Fix it everywhere